scripts/release_testing_setup.sh

```diff
@@ -25,8 +25,12 @@ git fetch --tags --quiet origin main
 git checkout main
 
 # The chunk below is to determine the latest version by searching
-# Get the latest released tag Cocoapods-X.Y.Z for release and prerelease testing, beta version will be excluded.
-test_version=$(git tag -l --sort=-version:refname --merged main 'CocoaPods-*[0-9]' | head -n 1)
+# Get the latest released tag Cocoapods-X.Y.Z for release and prerelease
+# testing, beta version will be excluded.
+# Note: If a long running release branch's tags are keeping the next tagged
+# release on main from being tested, add `--merged main` to the below git tag
+# command. Doing so will exclude tags that don't exist on main.
+test_version=$(git tag -l --sort=-version:refname 'CocoaPods-*[0-9]' | head -n 1)
```

Comment:
My preference would be to keep `--merged main` along with a suggestion that if the nightly tag isn't updated to check that the CocoaPods tag is on the main branch.

Reply:
I see, thanks. Done.